src/templates/_common/components/interview_popup.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+const InterviewPopup = () => (
+    <div id='interview_popup_container' className='invisible'>
+        <div className='popup'>
+            <div className='popup__head'>
+                <img className='popup__present_img' src={it.url_for('images/interview_popup/present.svg')} alt='$30 present' />
+                <h1>Do you have 40 minutes for an interview?</h1>
+            </div>
+            <div className='popup__body'>
+                <h2>Earn $30 to trade on binary.com</h2>
```

Comment:
please use something like `Earn $30 to trade on ${it.website_name.toLowerCase()}`